Eastern-Research-Group/mywaterway
# Feature/171 update survey mapping

## Related Issues:
* [HMW-171](https://jira.epa.gov/browse/HMW-171)

## Main Changes:
* Updated the `stateNationalUses.json` file.
* Updated the `surveyMapping.json` file.

## Steps To Test:
1. Navigate to http://localhost:3000/state/NH/water-quality-overview
2. Click on the `Aquatic Life` tab
3. Select `Lakes and Reservoirs` for the water type
4. Verify the pie chart is displayed
5. Navigate to http://localhost:3000/state/FL/water-quality-overview
6. Select `Lakes and Reservoirs` for the water type
7. Verify the pie chart is displayed
8. Select `Rivers and Streams` for the water type
9. Verify the pie chart is displayed
